sdk/src/androidTest/java/com/bugsnag/android/DeviceDataSummaryTest.java

```diff
@@ -63,6 +63,9 @@ public void testJsonSerialisation() throws JSONException {
         assertNotNull(deviceDataJson.getString("model"));
         assertNotNull(deviceDataJson.get("cpuAbi"));
         assertTrue(deviceDataJson.has("jailbroken"));
+
+        JSONObject versions = deviceDataJson.getJSONObject("runtimeVersions");
+        assertNotNull(versions.get("androidApiLevel"));
```

Comment:
is "not null" the best assertion you can make? Is this because the tests can run on any version of Android? Could this not be an int greater than `n`, where `n` is the minimum supported api level?

Reply:
I've changed this to >= 14, which is the lowest SDK version we support.